qa/pull-tester/rpc-tests.py

```diff
@@ -380,7 +385,7 @@ def get_next(self):
             log_stderr = tempfile.SpooledTemporaryFile(max_size=2**16)
             self.jobs.append((t,
                               time.time(),
-                              subprocess.Popen((self.tests_dir + t).split() + self.flags + port_seed,
+                              subprocess.Popen(eval(os.environ["CUSTOM_SCRIPT"]) + (self.tests_dir + t).split() + self.flags + port_seed,
```

Comment:
Even if we assume that variable cannot be attacker-controlled, it's impossible to review this change without knowing how `CUSTOM_SCRIPT` would be used.

Reply:
Added description and linked to script in comments.